cmd/upgrade.go

```diff
@@ -63,6 +69,7 @@ func NewUpgradeCmd() *cobra.Command {
 	f.StringVar(&uc.rawSubscriptionID, "subscription-id", "", "the subscription ID where the cluster is deployed")
 	f.StringVar(&uc.resourceGroupName, "resource-group", "", "the resource group where the cluster is deployed")
 	f.StringVar(&uc.deploymentDirectory, "deployment-dir", "", "the location of the output from `generate`")
+	f.StringVar(&uc.upgradeModelFile, "upgrademodel-file", "", "file path to upgrade API model")
```

Comment:
I was trying to be consistent. Its confusing to have deployment-dir and upgrade-model, no?